src/remove-task.js

```diff
@@ -0,0 +1,12 @@
+// Delete one task
+export default class Delete {
+  static deleteOne(arr, index) {
+    arr.splice(index, 1);
+    for (let i = 0; i < arr.length; i += 1) {
+      arr[i].index = i;
+    }
+    return arr;
+  }
```

Comment:
- [ ] After adding 5 tasks to your todo app, the indexes of those tasks were set to the value of the new array length (i.e. the index of the 5th task added to the list equal to 5 which is okay, however, after deleting a task, the index of those tasks should be set to the value of the new array i.e the index of the 4th task should be 4 but yours is 3 because the indexes of those tasks now started from 0 after deleting a task instead of 1. Kindly fix this such that when you remove a task, the index of the last task should equal the length of the array. See the image below for more understanding.

Add task | Remove task
:---:|:---:
![](https://i.imgur.com/9VoUpdz.png) | ![](https://i.imgur.com/QKClHtI.png)

Reply:
Thanks I had not noticed that, I have fixed it by adding `arr[i].index = i + 1;` indide the `for` loop